entries/b/bookbeat.fi.json

```diff
@@ -0,0 +1,17 @@
+{
+  "BookBeat": {
+    "domain": "bookbeat.fi",
+    "url": "https://www.bookbeat.fi/",
```

Comment:
```suggestion
```

The `url` attribute is only needed if going to the domain (e.g. `https://bookbeat.fi`) fails